vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Handling Usage Files
+description: Guidance for handling usage files.
+ms.date: 05/11/2023
+---
+
+# Handling Usage Files
+
+## Overview
+Usage files play an important role in providing instructions, examples, or guidelines for utilizing packages within our repository. To ensure consistency and clarity, we have established the following guidelines for handling usage files:
+
+**Using `configure_file()` Function**
+
+The recommended approach for handling usage files is by utilizing the `configure_file()` function provided by CMake. This function allows you to specify the source usage file and the destination path where it should be copied. 
+
+For example:
+
+```
+configure_file("${CMAKE_CURRENT_LIST_DIR}/usage" "${CURRENT_PACKAGES_DIR}/share/${PORT}/usage" COPYONLY)
+```
+
+This code snippet will copy the `usage` file located in the current directory to the destination path. `${CURRENT_PACKAGES_DIR}/share/${PORT}/usage`.
+
+**Naming Convention** 
```

Comment:
This section is unclear -- is it referring to the name inside `CURRENT_PACKAGES_DIR` or in `CURRENT_PORT_DIR`? It uses the phrase "packages directories", which sounds like `CURRENT_PACKAGES_DIR`.